DQM/SiPixelMonitorClient/src/SiPixelActionExecutor.cc

```diff
@@ -392,7 +392,7 @@ void SiPixelActionExecutor::fillSummary(DQMStore::IBooker& iBooker, DQMStore::IG
 		(*iv)=="size"||(*iv)=="sizeX"||(*iv)=="sizeY"||(*iv)=="minrow"||
 		(*iv)=="maxrow"||(*iv)=="mincol"||(*iv)=="maxcol")
 	  prefix="SUMCLU";
-          if(currDir.find("Track")!=string::npos) prefix="SUMTRK";
```

Comment:
This is very confusing. Is this meant to be included in  the previous if block  or is it starting a new if block? Same question for the rest of the changes.